tests/data/html/audio.html

```diff
@@ -1,11 +1,19 @@
-<figure><div><audio src=https://getfoobar/audio/sohtdbcmxu-1581683086.mp3></audio><div class="audio-player audio-player-disabled">
-    <div class="player-button gold-button"><i class="icon audio-play-icon"></i></div>
-    <div class="player-time player-current-time">00:00</div>
-    <div class="player-seekbar">
-        <div class="progress">
-            <div class="progress-pin"></div>
-            <div role="progressbar" aria-valuemin="0" aria-valuemax="100" class="progress-bar"></div>
-        </div>
-    </div>
-    <div class="player-time player-end-time">--:--</div>
+<figure><div><audio src=https://getfoobar/audio/sohtdbcmxu-1581683086.mp3></audio>  <div class="audio-player audio-player-initialising">
```

Comment:
it should be like

```
<figure class="audio-player-container">
  <audio ....
  ....
</figure>
```

Reply:
I will update the figure block to add this class.

Reply:
Done [Stack Audio Player](https://github.com/scrolltech/tiptapy/blob/af01bd1220988a1442f05a8f0ff4f562c01f34be/tiptapy/templates/stack-audio-player.html)